httpx/_transports/asgi.py

```diff
@@ -144,7 +148,7 @@ async def send(message: dict) -> None:
         try:
             await self.app(scope, receive, send)
         except Exception:
-            if self.raise_app_exceptions or not response_complete:
+            if self.raise_app_exceptions or not response_complete.is_set():
```

Comment:
I _think_ this was a typo before.

Reply:
Think that was my mistake. Missed this when changing `response_complete` from a bool to an event. Also didn't realise the `nonlocal` could be removed 😬